_posts/2021-10-07-open-source-economics-is-not-what-you-think.md

```diff
@@ -0,0 +1,70 @@
+
+As a result, when people hear the term "open source economics" they tend to jump to the same conclusions: it's about how 💵💶💷💴 flows between and is invested in open source projects. This is not a bad conclusion, it results in the creation of tools such as [GitHub Sponsors](https://github.com/sponsors) that benefit many open source maintainers but it's not the whole picture.
+
+![GitHub Sponsors]({{ '/images/a/github_sponsors.png' | absolute_url }})
+
+When you look just at the financial side of open source projects you tend to assume any problems they suffer are due to a lack of investment and can be solved by adding more 💵💶💷💴. For example, if a project struggles to make releases, review and merge pull requests, close issues or answer discussions: if they had more money they could pay someone to do that work and more of it will get done.
+
+Unfortunately, money alone will not always fix any of these problems. Money can help but it needs correct investment and understanding of what the _real_ problem is.
+
+### 👍 What are "Open Source Economics"?
+
+An "economic problem" is one which requires the allocation of limited resources in order to solve a problem. In modern, capitalist economies: the limited resources are usually 💵💶💷💴. Where there is a labour constraint, this is solved by paying enough money to compensate or attract the right candidates to solve the problem.
+
+Open source software is a little different because of the high degree of specificity, limited transferability/substitutability and non-financial incentives of "labour" (maintainers). Many widely used open source projects are built by a single maintainer and, if so, this maintainer is the only person who can make all releases, review and merge all pull requests. Even with multiple maintainers, a widely used project will have a much smaller number of people doing this work than consuming the results. This means that domain knowledge and access control is limited to this one person or limited group of people.
+
+[The Open Source Contributor Funnel](/2018/08/14/the-open-source-contributor-funnel-why-people-dont-contribute-to-your-open-source-project/) is pretty stark. In Homebrew's case: there's tens of maintainers, thousands of contributors and millions of users.
+
+![The Open Source Contributor Funnel]({{ '/images/a/the-open-source-contributor-funnel.png' | absolute_url }})
+
+Relatedly, the time that these maintainers spend on the project may not be optimally allocated. It's easy for their time to become monopolised by niche problems from a few users that require a lot of individual attention at the detriment of writing code to improve the project for the majority of users. This makes sense from the incentives of each user to just get their problem solved but not for the project as a whole (and may not even be the best for the user if it slows or halts feature development).
```

Comment:
Optimal for whom? The interests of the maintainer and the many users are not all the same.